docs/installation/registry-mirror.md

```diff
@@ -0,0 +1,104 @@
+The registry does not need to have any endpoints listed, it just needs to be present.
+For example, to enable distributed mirroring of images from `docker.io` and `registry.k8s.io`, configure `registries.yaml` with the following content on all cluster nodes:
+
+```yaml
+mirrors:
+  docker.io:
+  registry.k8s.io:
+```
+
+Endpoints for registry mirrors may also be added as usual.
+In the following configuration, images pull attempts will first try the embedded mirror, then `mirror.example.com`, then finally `docker.io`:
+```yaml
+mirrors:
+  docker.io:
+    endpoint:
+      - https://mirror.example.com
+```
+
+If no registries are enabled for mirroring on a node, that node does not participate in the distributed registry in any capacity.
+
+For more information on the structure of the `registries.yaml` file, see [Private Registry Configuration](./private-registry.md).
+
+### Disabling containerd Default Endpoints
+
+Nodes may be started with the `--disable-default-registry-endpoint` option.
+When this is set, containerd will not fall back to the default registry endpoint, and will only pull from configured mirror endpoints,
+and the distributed registry if it is enabled.
+
+This may be desired if your cluster is in a true air-gapped environment where the upstream registry is not available,
+or if you wish to have only some nodes pull from the upstream registry.
+
+## Security
+
+### Authentication
+
+Access to the embedded mirror's registry API requires a valid client certificate, signed by the cluster's client certificate authority.
+
+Access to the distributed hash table's peer-to-peer network requires a preshared key that is controlled by server nodes.
+Nodes authenticate each other using both the preshared key, and a certificate signed by the cluster certificate authority.
+
+### Potential Concerns
+
+Be aware that the embedded registry may make available images that a node may not otherwise have access to.
+For example, if some of your images are pulled from a registry, project, or repository that requires authentication via Kubernetes Image Pull Secrets, or credentials in `registries.yaml`,
+the distributed registry will allow other nodes to share those images without providing any credentials to the upstream registry.
+
+Users with access to push images into the containerd image store on one node may be able to use this to 'poison' the image for other cluster nodes,
+as other nodes will trust the tag advertised by the node, and use it without checking with the upstream registry.
+If image integrity is important, you should use image digests instead of tags, as the digest cannot be poisoned in this manner.
+
+The distributed registry is built on peer-to-peer networking, and assumes an equal level of privilege and trust between all cluster members.
+If this does not match your cluster's security posture, you should not enable the embedded distributed registry.
```

Comment:
I would consider wrapping this in a :::warning admonition, just to get the TLDR point across.